src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -88,7 +88,7 @@
 	
 	private static final ArgsMessage m_reloading = new ArgsMessage(CONFIG_NODE + ".reload.reloading");
 	
-	private static void reloading(CommandSender sender, String what, Object... args) {
+	private static void reloading(CommandSender sender, String what, final Object... args) {
```

Comment:
Any specific reason for this change?

Reply:
That's definitely unintended, will revert that